### ISSUE TITLE
ORCID

### ISSUE THREAD

**Issue by asdfjkl** [Cryptosaurus/iacrtrans#17]
It seems there is currently no dedicated command to include an author's orcID. As this becomes more and more important to avoid name confusions, a command like in Springer's lncs (\orcid) would be useful.

**Comment by MrPugh** [comment]
In my personal opinion, email addresses due to their unique nature should be quite sufficient to resolve name confusions.

**Comment by asdfjkl** [comment]
email addresses often change once you change your employer. If you are a postdoc building up your reputation, and there is another person with the same name, this will lead to problems. That's what orcID is for.

**Comment by MrPugh** [comment]
I agree that a more 'permanent' email address than an affiliation address is required.

I am wondering how long ORCID is going to be around; maybe the next big thing is author IDs in the blockchain...

**Comment by asdfjkl** [comment]
I've added a pull-request https://github.com/Cryptosaurus/iacrtrans/pull/31
Unfortunately, this repo does not seem to be actively maintained...

**Comment by kmccurley** [comment]
I believe ORCID may outlive both SMTP and LaTeX. 😬 I've been working on a new LaTeX cls file the new IACR journal, and we will definitely ask for ORCID IDs - they are incredibly important for people to get credit for their work in ranking algorithms by Clarivate, SCOPUS, and Google Scholar. See [our paper](https://arxiv.org/pdf/2301.08277.pdf) on the subject.
